src/Microsoft.Management.Configuration/Microsoft.Management.Configuration.idl

```diff
@@ -395,6 +410,13 @@ namespace Microsoft.Management.Configuration
         IApplySettingsResult ApplySettings();
     }
 
+    [contract(Microsoft.Management.Configuration.Contract, 2)]
+    interface IConfigurationUnitProcessor2 requires IConfigurationUnitProcessor

```

Comment:
###### Sorry for not seeing this earlier...
Maybe this should be an explicit interface for `GetAllSettings`, rather than an interface in the generic evolution of the unit processor.  That way we can use it at the orchestration layer to know that export isn't expected to work rather than calling and having each processor need to give us the exact same error.

Reply:
Renamed the interface and made the unit processor here not implement it anymore.